Базовая стилизация всех страниц

comeunne opened the pull request:
---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1755265/tasks/6)

comeunne committed:
Исправляет иконку гамбургера и отступы

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,115 @@
+<body>
+  <header class="page-header">
+    <nav class="navigation">
+      <a class="navigation-logo" href="index.html"><img src="images/logo-sedona.svg" alt="Логотип Седоны" width="139"
```
Не должно быть ссылки на текущую страницу. Удали href

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,115 @@
+      <a class="navigation-logo" href="index.html"><img src="images/logo-sedona.svg" alt="Логотип Седоны" width="139"
+          height="70"></a>
+      <ul class="navigation-list">
+        <li class="navigation-item"><a class="navigation-link navigation-link-current" href="index.html">Главная</a>
```
И здесь href удали

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,115 @@
+  <main class="main-index"><span class="visually-hidden">Добро пожаловать в Седону</span>
+    <h2 class="welcome-index-title">Седона - небольшой городок в Аризоне, заслуживающий большего!</h2>
+    <p class="advantage-index-title">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Канон!</p>
+  </main>
```
main должен оборачивать все секции и тянуться вплоть до футера

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,115 @@
+</head>
+
+<body>
+  <header class="page-header">
```
В Седоне весь контент сайта ограничен по ширине, поэтому все (header, main, footer) нужно обернуть в блок-обертку. В итоге должна быть такая структура
body
--page-wrapper
----header
----main
----footer

comeunne replied:
вложил все (header, main, footer) в body class="page-wrapper"

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,115 @@
+      </ul>
+    </nav>
+  </header>
+  <main class="main-index"><span class="visually-hidden">Добро пожаловать в Седону</span>
```
Зачем нужен тут скрытый спан?

Muhammadabumansur replied:
![image](https://user-images.githubusercontent.com/16342548/154526704-8a31953f-2d21-4bcb-91b2-965f5d4b7ba5.png)

Вот это отдельная секция. Я не вижу ее в разметке

comeunne replied:
вложил эту секцию в section class="main-index-background"

Muhammadabumansur replied:
Что это значит? Вижу что ты зачем-то обернул header в секцию. Это замечание не касалось хедера и его не нужно оборачивать в секцию. Не делай наугад, если не понимаешь смысл замечания, лучше спроси сначала. Тебе просто нужно добавить эту секцию. Пока можешь оставить пустой Потом ты добавишь в нее картинку (надпись) и задашь фон

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,115 @@
+        <p class="advantage-number">- №5 -</p>
+        <p class="advantage-description">Большинство едет в Гранд-Канон и толпится там</p>
+      </li>
+    </ul>
```
Здесь неправильная структура. Надо поделить так.
![image](https://user-images.githubusercontent.com/16342548/154529030-e64853f5-afcd-4b83-ac80-ea10b71b5beb.png)

Внешний список нумерованный ol

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,115 @@
+    <address class="phone-logo-academy">
+      <a class="footer-phone-number" href="tel:+78128121212">+7 (812) 812-12-12</a>
+      <a href="https://htmlacademy.ru/"><span class="visually-hidden">Логотип Академии</span></a>
+    </address>
```
Тут address не нужно ставить, просто две ссылки

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,175 @@
+      <ul class="navigation-list">
+        <li class="navigation-item"><a class="navigation-link" href="index.html">Главная</a></li>
+        <li class="navigation-item"><a class="navigation-link" href="#">О Седоне</a></li>
+        <li class="navigation-item"><a class="navigation-link navigation-link-current" href="catalog.html">Гостиницы</a>
```
Здесь тоже страница ссылается на себя

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,175 @@
+    <header class="inner-header">
+      <h1 class="inner-header-title">Гостиницы Седоны</h1>
+      <ul class="breadcrumbs">
+        <li class="breadcrumbs-item"><span class="visually-hidden">Иконка домика</span></li>
```
Это ссылка

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,175 @@
+      <h1 class="inner-header-title">Гостиницы Седоны</h1>
+      <ul class="breadcrumbs">
+        <li class="breadcrumbs-item"><span class="visually-hidden">Иконка домика</span></li>
+        <li class="visually-hidden"><span class="vissualy-hidden">Стрелка направо</span></li>
```
Не нужен здесь текст "Стрелка направо", он не передает никакого нужного смысла. Этот элемент вообще можно удалить и использовать псевдоэлемент в css

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,175 @@
+        <div class="hotel-filter-price">Стоимость, ₽:</div>
+        <span class="visually-hidden">Ползунок регулирования цены</span>
+      </fieldset>
+      <a class="button-apply" href="#">Применить</a>
+      <p class="button-reset">Сбросить</p>
```
Это кнопки. Одна submit а другая reset

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,175 @@
+      <option value="expensive">Сначала дорогое</option>
+      <option value="popular">Сначала популярное</option>
+    </select>
+    <span class="visually-hidden">Режим: плитка</span>
+    <span class="visually-hidden">Режим: слайд-шоу</span>
+    <span class="visually-hidden">Режим: список</span>
```
Тут должен быть список и это интерактивные элементы

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,175 @@
+    <span class="visually-hidden">Режим: слайд-шоу</span>
+    <span class="visually-hidden">Режим: список</span>
+  </section>
+  <section class="sorting-filter-results">
+    <ul class="hotel-card-list">
```
У этой секции нет заголовка

comeunne replied:
добавил -span class="vissualy-hidden"-span в остальные секции тоже

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,175 @@
+        <img class="hotel-card-image-1" src="images/hotel-1.jpg" alt="Отель Амара Резорт и Спа" width="300"
+          height="206">
+        <h3 class="hotel-card-title">Amara Resort & Spa</h3>
+        <p class="hotel-card-type hotel-card-price">Гостиница</p>
+        <p class="hotel-card-price">От 4000 ₽</p>
```
Эти два элемента лучше объединить в один, чтоб легче было разложить в ряд

comeunne replied:
вложил в -span class="hotel-card-type-price">Гостиница От 4000 ₽-span

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,175 @@
+        <h3 class="hotel-card-title">Amara Resort & Spa</h3>
+        <p class="hotel-card-type hotel-card-price">Гостиница</p>
+        <p class="hotel-card-price">От 4000 ₽</p>
+        <a class="hotel-card-button-left" href="#">Подробнее</a>
+        <a class="hotel-card-button-right" href="#">В избранное</a>
```
Эти два элемента тоже лучше объединить в один, чтоб легче было разложить в ряд. И "В избранное" это кнопка

comeunne replied:
вложил в span class="hotel-card-button-left">Подробнее <button class="hotel-card-button-right" type="button">В избранное</button

comeunne replied:
вложил в -span class="hotel-card-button-left"-Подробнее -button class="hotel-card-button-right" type="button"-В избранное-button--span

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,175 @@
+        <p class="hotel-card-price">От 4000 ₽</p>
+        <a class="hotel-card-button-left" href="#">Подробнее</a>
+        <a class="hotel-card-button-right" href="#">В избранное</a>
+        <a href="#" class="first-star"></a>
+        <a href="#" class="second-star"></a>
+        <a href="#" class="third-star"></a>
+        <a href="#" class="fourth-star"></a>
```
Это не интерактивные элементы, тут можно оставить просто один div а звезды будут фоном. И этот div нужно объединить с рейтингом в один блок

comeunne replied:
добавил -div class="stars-underground-3"--div class="hotel-rating"-Рейтинг: 8,9--/div--/div- 
число stars-underground соответствует количеству звезд

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,175 @@
+      <li class="pagination-item"><a class="pagination-link" href="#">...</a></li>
+      <li class="pagination-item"><a class="pagination-link" href="#">10</a></li>
+    </ul>
+    <p class="hotel-number-show-page">Гостиниц на странице:</p>
+    <select class="select-control" aria-label="Количество отображаемых страниц">
+      <option value="four">4</option>
+      <option value="ten">10</option>
+      <option value="twenty">20</option>
+      <option value="fourty">40</option>
+    </select>
```
Это лучше объединить

Muhammadabumansur replied:
Объединить это значит что p и select должны быть вместе в одном общем блоке. Это не значит что select Надо переместить внутрь p

comeunne replied:
Я пробовал так, p внутрь select, перед первым option, валидатор ругается

Muhammadabumansur reviewed (commented):
Прежде чем переходить к стилям, нужно поправить разметку. У тебя много замечаний по разметке. Исправляй все, позже по стилям тоже замечания отправлю

comeunne committed:
Первые исправления

comeunne committed:
Конечные исправления

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,119 @@
-         <li class="navigation-item"><a class="navigation-link" href="catalog.html">Гостиницы</a></li>
-       </ul>
+      <nav class="navigation">
+        <a class="navigation-logo" href="#"><img src="images/logo-sedona.svg" alt="Логотип Седоны" width="139"
```
Чуть внимательнее. Я не писал указать в href #, я написал что надо вообще удалить атрибут href везде где ссылка на текущую страницу

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,119 @@
+</head>
+
+<body class="page-wrapper">
+  <section class="main-index-background">
```
Неправильно. Ты вложил только header. А нужно все (header, main, footer) И тут не должно быть section. Это просто обертка, у нее нет секционной семантики

Muhammadabumansur replied:
И не забывай про cataog.html

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,119 @@
-                   <label for="mailing-email">Email</label>
-                    <input class="field" placeholder="Ваш e-mail" type="email" name="mailing-email" id="mailing-email" required>
-                     <button class="button mailing-button" type="submit">Подписаться</button>
-                  </form>
-               </section>
-                <footer class="footer-contacts">
-                 <ul class="footer-social-list">
-                   <li class="footer-social-item"><a class="footer-button" href="https://twitter.com/htmlacademy_ru"><span class="visually-hidden">Twitter</span></a></li>
-                   <li class="footer-social-item"><a class="footer-button" href="https://www.facebook.com/htmlacademy"><span class="visually-hidden">Facebook</span></a></li>
-                   <li class="footer-social-item"><a class="footer-button" href="https://www.youtube.com/c/HTMLAcademyRUS"><span class="visually-hidden">Youtube</span></a></li>
-                 </ul>
-                  <address class="phone-logo-academy">
-                   <a class="footer-phone-number" href="tel:+78128121212">+7 (812) 812-12-12</a>
-                    <a href="https://htmlacademy.ru/"><span class="visually-hidden">Логотип Академии</span></a>
-                  </address>
-                </footer>
-   </body>
- </html>
+          <h3 class="advantage-title">Там есть мост дьявола</h3>
+          <p class="advantage-number">- №2 -</p>
+          <p class="advantage-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+        </li>
+      </ul>
+      <ol class="ordered-list-advantage">
+        <li class="advantage-item-3">
+          <h3 class="advantage-title">Небольшая площадь</h3>
+          <p class="advantage-number">- №3 -</p>
+          <p class="advantage-description">Все достопримечательности находятся очень близко</p>
+        </li>
+        <li class="advantage-item-4">
+          <h3 class="advantage-title">Красивая дорога</h3>
+          <p class="advantage-number">- №4 -</p>
+          <p class="advantage-description">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+        </li>
+        <li class="advantage-item-5">
+          <h3 class="advantage-title">Мало туристов</h3>
+          <p class="advantage-number">- №5 -</p>
+          <p class="advantage-description">Большинство едет в Гранд-Канон и толпится там</p>
+        </li>
+      </ol>
```
Я же скинул картинку - схему как надо разметить. На картинке совсем другая структура. У тебя должен быть OL внутри которого 5 LI а внутри первого LI вложенный UL из 3 LI. Внимательно смотри на картинку.

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,177 @@
+      <h1 class="inner-header-title">Гостиницы Седоны</h1>
+      <ul class="breadcrumbs">
+        <li class="breadcrumbs-item"><a class="little-house-icon"></a></li>
+        <li class="breadcrumbs-item"></li>
```
Тут должен быть псевдоэлемент, мы же это обсуждали. Удали этот LI

comeunne replied:
хотел как вариант псевдоклассом second-child и в него добавить эту стрелку, это неверно?
или можно например так: breadcrumbs:first-child::after, как думаешь?

Muhammadabumansur replied:
Там надо делать так - у каждого li кроме первого вставлять псевдоэлемент перед ним. Но пока это не нужно

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,177 @@
+        <div class="hotel-filter-price">Стоимость, ₽:</div>
+        <span class="visually-hidden">Ползунок регулирования цены</span>
+      </fieldset>
+      <button class="button-apply" type="submit">Применить</button>
+      <button class="button-reset" type="reset">Сбросить</button>
```
Эти кнопки объедини в один общий блок, иначе ты их не сможешь расположить как в макете

comeunne replied:
обернул эти кнопки в div class="button-apply-reset" или лучше по-другому?

Muhammadabumansur replied:
Ок. Нормальные названия классам научишься давать с опытом)

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,177 @@
+        <option value="expensive">Сначала дорогое</option>
+        <option value="popular">Сначала популярное</option>
+      </select>
+      <ul class="display-mode-catalog">
+        <li class="display-mode-catalog-1"><input class="tile-mode" type="image" src="images/mode-tile.svg" width="16"
+            height="14" alt="Режим отображения: плитка"></li>
+        <li class="display-mode-catalog-2"><input class="slideshow-mode" type="image" src="images/mode-slideshow.svg"
+            width="16" height="14" alt="Режим отображения: плитка"></li>
+        <li class="display-mode-catalog-3"><input class="list-mode" type="image" src="images/mode-list.svg" width="16"
+            height="14" alt="Режим отображения: список"></li>
+      </ul>
```
Это просто кнопки

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,177 @@
+      <button class="button-reset" type="reset">Сбросить</button>
+    </form>
+    <section class="sorting-filer">
+      <span class="visually-hidden">Список отелей</span>
```
span не имеет семантики, для заголовка используй h2, исправь это везде

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,177 @@
+        <li class="hotel-card-item">
+          <img class="hotel-card-image-2" src="images/hotel-2.jpg" alt="Отель Дезерт Кали" width="300" height="206">
+          <h3 class="hotel-card-title">Desert Quail Inn</h3>
+          <span class="hotel-card-type-price">Гостиница От 3000 ₽</span>
```
Не так. Это два отдельных элемента "Гостиница" и "От 3000 ₽". Но их нужно вложить в один общий (div или p)

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,177 @@
+          <img class="hotel-card-image-2" src="images/hotel-2.jpg" alt="Отель Дезерт Кали" width="300" height="206">
+          <h3 class="hotel-card-title">Desert Quail Inn</h3>
+          <span class="hotel-card-type-price">Гостиница От 3000 ₽</span>
+          <span class="hotel-card-button-left">Подробнее <button class="hotel-card-button-right" type="button">В
```
Тут что-то странное. Надпись "Подробнее" должна быть ссылкой, как ты сейчас нажмешь на нее? У тебя там должны быть ссылка и кнопка внутри одного общего блока. Лучше div или p

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,177 @@
+          <span class="hotel-card-type-price">Гостиница От 3000 ₽</span>
+          <span class="hotel-card-button-left">Подробнее <button class="hotel-card-button-right" type="button">В
+              избранное</button></span>
+          <div class="stars-underground-2">
+            <div class="hotel-rating">Рейтинг: 9,2</div>
+          </div>
```
Ок, хорошо что добавил блок stars-underground-2, но зачем рейтинг вставлять внутрь него? Это же два отдельных элемента

Muhammadabumansur commented:
В стилях если значение 0, то не нужно указывать единицу измерения. Пример -

![image](https://user-images.githubusercontent.com/16342548/154828850-6a8264c8-e259-4f16-b4b2-2efd10ed2ad6.png)

Muhammadabumansur commented:
![image](https://user-images.githubusercontent.com/16342548/154828874-467b0f79-cd58-4c51-a159-981d5712519a.png)

Одно и то же правило повторяется

Muhammadabumansur commented:
![image](https://user-images.githubusercontent.com/16342548/154828881-23cc67ed-d0ee-4a0a-950e-ec99a0a7f993.png)

![image](https://user-images.githubusercontent.com/16342548/154828937-b7cf47c8-45f2-4937-a223-1ab5e81b4ef9.png)


Фиксированную высоту не надо указывать

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,119 @@
+          <li class="navigation-item"><a class="navigation-link navigation-link-current" href="#">Главная</a>
+          </li>
+          <li class="navigation-item"><a class="navigation-link" href="#">О Седоне</a></li>
+          <li class="navigation-item"><a class="navigation-link" href="#">Гостиницы</a></li>
```
Здесь не надо было удалять ссылку на каталог

Muhammadabumansur commented:
![image](https://user-images.githubusercontent.com/16342548/154828965-f1d03bc1-a3fe-4880-909b-301eba2a0eea.png)

Не заданы стили

comeunne committed:
Исправляет

Исправляет кнопки, каталог с преимуществами, исправления href и тд

comeunne committed:
оставляет один ol

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,131 @@
+  <link rel="stylesheet" href="styles/styles.css">
+</head>
+
+<body class="page-wrapper">
+  <header class="page-header">
+    <nav class="navigation">
```
Так и не сделал обертку для всей страницы

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,131 @@
+            </li>
+          </ul>
+        </li>
+        <li class="aside-image"><img src="images/city-sedona.jpg" alt="Фото города Седона" width="800" height="384">
+        </li>
+        <li class="aside-image"><img src="images/devil-bridge.jpg" alt="Мост дьявола" width="800" height="384"></li>
```
Эти картинки не должны быть отдельными элементами списка. Смотри на макет, там элементы пронумерованы от 1 до 5. И значит в разметке должно быть 5 элементов. На картинке это соответствует красным прямоугольникам. Вот дублирую картинку

![image](https://user-images.githubusercontent.com/16342548/154894220-22f6c183-da6f-415b-b7ab-9683af6d0a07.png)

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,131 @@
+        </li>
+      </ol>
+    </section>
+    <section class="hotel-search">
+      <h2 class="visually-hidden">Поиск отеля в Седоне</span>
+        <h2 class="search-title">Заинтересовались?</h2>
```
Зачем это? Ты пытаешься вставить два h2? Нужен один. И у тебя невалидный код. Открывающий тег h2 а закрывающий span. Проверь весь свой код в валидаторе. И всегда проверяй в валидаторе прежде чем отправить на проверку

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,131 @@
+        </form>
+    </section>
+  </main>
+  <footer class="footer-contacts">
+    <ul class="footer-social-list">
+      <li class="footer-social-item"><a class="footer-button" href="https://twitter.com/htmlacademy_ru">
+          <h2 class="visually-hidden">Twitter</h2>
+        </a></li>
+      <li class="footer-social-item"><a class="footer-button" href="https://www.facebook.com/htmlacademy">
+          <h2 class="visually-hidden">Facebook</h2>
+        </a></li>
+      <li class="footer-social-item"><a class="footer-button" href="https://www.youtube.com/c/HTMLAcademyRUS">
+          <h2 class="visually-hidden">Youtube</span>
+        </a></li>
+    </ul>
+    <a class="footer-phone-number" href="tel:+78128121212">+7 (812) 812-12-12</a>
+    <a href="https://htmlacademy.ru/">
+      <h2 class="visually-hidden">Логотип Академии</h2>
+    </a>
```
Социальные кнопки и логотип академии это заголовки?

comeunne replied:
ты писал про семантику (смысл) в прошлых комментах,  и праильно будет использовать h2. поэтому добавил h2 и для лучшей читаемости кода. но я не до конца понимаю значение и смысл использования visually-hidden в коде

Muhammadabumansur replied:
visually-hidden так принято называть класс, для которого прописывают такие стили, чтоб элемент был визуально скрыт но оставался доступен для скринридеров

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,195 @@
+          <img class="hotel-card-image-1" src="images/hotel-1.jpg" alt="Отель Амара Резорт и Спа" width="300"
+            height="206">
+          <h3 class="hotel-card-title">Amara Resort & Spa</h3>
+          <div class="hotel-card-type">Гостиница<p class=«hotel-card-price»>От 4000 ₽</p>
```
`Гостиница` это один элемент а `От 4000 ₽` это другой элемент

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,195 @@
+          <div class="hotel-card-buttons"><a class="hotel-card-button-left">Подробнее</a><button
+              class="hotel-card-button-right" type="button">В избранное
+            </button></div>
+          <div class="stars-underground-4">
+            Рейтинг: 8,5
+          </div>
```
У тебя должен быть отдельный элемент для звезд и отдельный элемент для рейтинга

Muhammadabumansur commented:
В стилях если значение 0, то не нужно указывать единицу измерения - не исправлено

Muhammadabumansur commented:
Советую тебе внимательно читать замечания, по некоторым замечаниям ты либо не исправляешь то, что написано, либо делаешь что-то другое. Если непонятно - спрашивай

comeunne committed:
Исправляет последние замечания

все единицы 0px -> 0, ограничивает ширину документа оберткой, гостиница и цена разделены классами, звезды и рейтинг разделены классами, убраны-добавлены скрытые h2 заголовки, разметка преимуществ изменена.

comeunne committed:
Выравнивает отступы

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,135 @@
+            <img class="aside-image" src="images/city-sedona.jpg" alt="Фото города Седона" width="800" height="384">
+            <ul>
+              <li class="tips-item-1">
+                <h2 class="visually-hidden">Иконка-жильё</h2>
```
Эти все иконки сделай просто span или div

comeunne replied:
сделал span

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,135 @@
+        </ol>
+      </section>
+      <section class="hotel-search">
+        <h2 class="visually-hidden">Поиск отеля в Седоне</h2>
+        <h2 class="search-title">Заинтересовались?</h2>
```
Два заголовка h2, убери скрытый

Muhammadabumansur commented on index.html:
```diff
@@ -1,74 +1,135 @@
+      </ul>
+      <a class="footer-phone-number" href="tel:+78128121212">+7 (812) 812-12-12</a>
+      <a href="https://htmlacademy.ru/">
+        <h2 class="visually-hidden">Логотип Академии</h2>
```
Тут не должен быть заголовок

Muhammadabumansur commented on catalog.html:
```diff
@@ -1,157 +1,209 @@
+      </form>
       <section class="sorting-filer">
-       <h2 class="hotel-search hotel-search-results">Найдено гостиниц: 38</h2>
+        <h2 class="visually-hidden">Список отелей</h2>
+        <h2 class="hotel-search hotel-search-results">Найдено гостиниц: 38</h2>
```
Тут тоже два h2, так не должно быть

Muhammadabumansur commented:
Я приму задание, так как мы очень долго с ним возимся. Но надо все поправить
